tests/conftest.py

```diff
@@ -111,6 +111,8 @@ def pytest_addoption(parser):
                     help="Change default loops delay")
     parser.addoption("--logs_since", action="store", type=int,
                     help="number of minutes for show techsupport command")
+    parser.addoption("--collect_techsupport", action="store", default=False, type=bool,
```

Comment:
As much as I doubt anyone is using all these dumps at the moment, do you think we should set the default to enable just to keep it consistent with the current behavior?

Reply:
we certainly can do that. Then, the nightly run would disable it until we reach stable baseline. @daall, @yxieca what do you guys think?